lib_email/src/test/resources/schema.sql

```diff
@@ -0,0 +1,21 @@
+--Only create the tables we care about.
+
+-- -----------------------------------------------------
+-- Table `users`
+-- -----------------------------------------------------
```

Comment:
Not sure what happened to "confirm_token" table which was being used to confirm the user account

Reply:
confirm_token was taken out of auth library because it's outside of the scope of what it's meant to do (it doesn't verify users, that's for users ms and lib_email to deal with)